src/app/dashboard/activities/list-activities/chart-statistics/chart.component.html

```diff
@@ -1,4 +1,8 @@
 <div class="ps-chart-wrapper" *ngIf="selectedUsers.length > 0 && parentHasActivity">
+  <div>
+    <input type="radio" value="Group" [checked]="chartType === 'group'" (change)="changeChart('group')"> Group
```

Comment:
Please use Bulma radio styles (If we have this feature in new UX then apply it in the new UX branch)